v1/plugin/integer.go

```diff
@@ -19,31 +19,15 @@ limitations under the License.
 
 package plugin
 
-import "fmt"
+import "github.com/intelsdi-x/snap-plugin-lib-go/v1/plugin/rpc"
 
-// integerRule defines a type to contain Integer specific rule data.
-type integerRule struct {
-	Key string
-
-	Default    int64
-	HasDefault bool
-
-	Required bool
-
-	Minimum int64
-	HasMin  bool
-
-	Maximum int64
-	HasMax  bool
-}
-
-type integerRuleOpt func(*integerRule)
+type integerRuleOpt func(*rpc.IntegerRule)
 
 // SetDefaultInt Allows easy setting of the Default value for an integerRule.
 // Usage:
 //	//	NewIntegerRule(key, req, config.SetDefaultInt(default))
```

Comment:
Need to modify these comments that reference functions that don't exists anymore.